homeassistant/components/ohme/manifest.json

```diff
@@ -7,5 +7,5 @@
   "integration_type": "device",
   "iot_class": "cloud_polling",
   "quality_scale": "silver",
-  "requirements": ["ohme==1.3.0"]
```

Comment:
Worth splitting the dependency change out to another PR?

Reply:
This one is in #139013 - I should've put that in the description :) 

I'll merge dev back into this to hide the change though.